modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceconfiguration/update/ServiceConfigurationChangeDetailsEntity.java

```diff
@@ -79,4 +80,9 @@ public class ServiceConfigurationChangeDetailsEntity implements Serializable {
     @Column(name = "STATUS")
     private ServiceConfigurationStatus status;
 
+    @Column(name = "TASK_RESULT ", columnDefinition = "json")
+    @Type(value = JsonType.class)
+    @Convert(converter = ObjectJsonConverter.class)
+    private ServiceConfigurationChangeResult taskResult;
```

Comment:
we must only store the 'tasks' from the  ServiceConfigurationChangeResult object. Not the complete ServiceConfigurationChangeResult object. 
From the ServiceConfigurationChangeResult oject, the main error and status is already stored in other columns.